src/base/utils.js

```diff
@@ -63,6 +63,12 @@ export const Fullscreen = {
       document.msFullscreenElement
     )
   },
+  getFullscreenElement: function() {
```

Comment:
Since you're adding the `getFullscreenElement` helper, the `isFullscreen` function should use it to avoid repeated code.